frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -180,6 +180,29 @@ action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected br
 
 # Graph Table
 
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.commit=commit
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.fork-point=fork point ?
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.found-in-reflog=has been found in reflog of
+
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.applying=APPLYING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.bisecting=BISECTING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.cherry-picking=CHERRY-PICKING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.merging=MERGING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.rebasing=REBASING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.reverting=REVERTING
+
+string.GitMachete.BranchOrCommitCellRendererComponent.stps-tooltip.in-sync=Branch ''{0}'' is in sync with ''{1}''
```

Comment:
`stps` and `strs` are **inconsistent** with:

`action.GitMachete.ISyncToParentStatusDependentAction.description.sync-to-parent-status.in-sync=in sync with its parent`

and 

`action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.ahead-of-remote=ahead of its remote`

Reply:
Done